app/views/users/_user_list.html.erb

```diff
@@ -6,7 +6,7 @@
                 <%= presenter.gravatar_image(size: 80,
                                              class: 'img-rounded media-object pull-left') %>
                 <div class="media-body index-summary">
-                  <p class="media-heading"><%= presenter.display_name %>
+                  <p class="media-heading" title="<%= presenter.display_name unless presenter.display_name.length < 15 %>"><%= truncate(presenter.display_name, length: 14) %>
```

Comment:
this feels like too much logic in the view - either pull to a helper, or check if you even need the `unless` statement